frontend/src/components/TaskCreator/GroupMemberPicker.tsx

```diff
@@ -33,8 +33,29 @@ export default function GroupMemberPicker({
   };
   const reset = (): void => {
     onMemberChange(undefined);
-    if (clearAssignedMember) clearAssignedMember();
+    if (clearAssignedMembers) clearAssignedMembers();
   };
+  const getMemberNames = (selectedMembers: readonly SamwiseUserProfile[]): string => {
+    let memberNames = '';
+    let i = 0;
+    while (memberNames.length < 12 && i < selectedMembers.length) {
+      const firstName = selectedMembers[i].name.split(' ')[0];
+      if (memberNames.length + firstName.length < 12) {
```

Comment:
why number 12?

Reply:
It's to limit the amount of characters displayed on the 'assign to' button.